src/orders/model/OrderCreateRequest.ts

```diff
@@ -0,0 +1,26 @@
+import LineItem from './LineItem';
+import Fulfillment from './Fulfillment';
+import AdditionalField from '../../model/AdditionalField';
+import OrderStatus from './OrderStatus';
+import Customer from './Customer';
+
+export default interface OrderCreateRequest {
+  additionalFields?: AdditionalField[];
+  channelOrderId: string;
+  channelId: string;
+  businessId: string;
```

Comment:
businessId is derived from channelId when an order is created. This parameter will not do anything. Please remove it.